16_00_cct.Rmd

```diff
@@ -264,22 +264,23 @@ Hugh Cole has been director of Policy and Strategy at the City of Cape Town sinc
 
 [Derek Strong](https://drkrynstrng.gitlab.io/) is currently a Research Computing Associate with the Center for Advanced Research Computing at the University of Southern California. Prior to this, he worked with Kelsey Jack at UC Santa Barbara and with the City of Cape Town to identify and implement solutions for streamlining municipal data sharing with researchers. His interests include research computing, open science, and meta-research.
 
-[Brendan Maughan-Brown](https://sites.google.com/site/bmaughanbrown/) is an interdisciplinary social scientist with expertise on: the uptake of HIV-prevention and treatment services; behavioural economics; and the social and behavioural determinants of HIV risk. He is a Chief Research Officer at the Southern Africa Labour and Development Research Unit, University of Cape Town.
+[Brendan Maughan-Brown](https://sites.google.com/site/bmaughanbrown/) is an interdisciplinary social scientist with expertise on: the uptake of HIV-prevention and treatment services; behavioral economics; and the social and behavioral determinants of HIV risk. He is a Chief Research Officer at the Southern Africa Labour and Development Research Unit, University of Cape Town.
 
 ## Acknowledgements {-}
 
 The authors thank Gordon Inggs, Yogini Jivanji, Craig Kesson, Kayleen Simpson, Carol Wright, Joshua Hawley, and the IDEA team at J-PAL for support of the collaboration and for constructive comments on the draft.
 
-[^cct6]: A summary of implementation activities associated with the Data Strategy is provided at http://osf.io/2a7ev. The document is not yet publicly available but will be provided at this same location when it becomes available.
+[^cct6]: A summary of implementation activities associated with the Data Strategy is provided at http://osf.io/2a7ev (accessed 2020-12-11). The document is not yet publicly available but will be provided at this same location when it becomes available.
```

Comment:
The document should be cited, not URL. The extra sentence should be stricken.

Reply:
The summary they link to is actually duplicated in appendix A, so I'll change it to that.